src/libs/ajax.js

```diff
@@ -1168,7 +1168,8 @@ const Apps = signal => ({
           diskConfig: {
             name: diskName,
             labels: {
-              saturnApplication: 'galaxy'
+              saturnApplication: 'galaxy',
+              workspaceName
```

Comment:
This seems like a separate change, to me? From my understanding, this is passing the workspaceName as a label to Leo? Is that related to the workspace v2 parameters?

Reply:
Yes, it is part of the ticket https://broadworkbench.atlassian.net/browse/IA-2410